plugins/polio/models.py

```diff
@@ -372,9 +372,6 @@ class Meta:
     preperadness_spreadsheet_url = models.URLField(null=True, blank=True)
     # DEPRECATED -> Moved to round.
     preperadness_sync_status = models.CharField(max_length=10, default="FINISHED", choices=PREPAREDNESS_SYNC_STATUS)
-    # Surge recruitment. Not really used anymore
-    surge_spreadsheet_url = models.URLField(null=True, blank=True)
-    country_name_in_surge_spreadsheet = models.CharField(null=True, blank=True, max_length=256)
```

Comment:
To double-check with @madewulf or @mathvdh but it seems to me @olethanh 's policy with this model is to deprecate fields but not delete them

Reply:
In this case, I think we can really delete, but we have to include the corresponding migration

Reply:
@quang-le , @madewulf there are three migrations which have been created to remove the fields: